daemon/api.go

```diff
@@ -283,8 +284,8 @@ func getPackagesInfo(c *Command, r *http.Request) Response {
 
 	for _, v := range bags {
 		m := v.Map(nil)
-		name := m["name"]
-		origin := m["origin"]
+		name, _ := m["name"].(string)
```

Comment:
That on itself sounds fine, but the construction here looks slightly curious indeed. I'd expect an error returned at some point, rather than accepting the bogus data as fine.